### ISSUE TITLE
Change default plot file format to PNG

### ISSUE THREAD

**Issue by asoplata** [DynaSim/DynaSim#460]
tldr: The default plot file format is SVG. I think the default plot output format of DS should use PNG, since it doesn't have compression artifacts like JPG, and "high-complexity" data/visualizations like membrane potential time series can make SVGs become VERY large.

Here's some good sites illustrating the differences between image file formats, and guidelines on when to use which type:

https://photo.stackexchange.com/questions/19270/what-are-jpeg-artifacts-and-what-can-be-done-about-them

https://stackoverflow.com/questions/2336522/png-vs-gif-vs-jpeg-vs-svg-when-best-to-use

https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization#selecting-the-right-image-format

In day-to-day simulation and plot printing, using PNGs saves on space and I think is detailed enough for almost all of our visualization. When it comes time to send figures to the publisher, even though vectors like EPS or SVG are large, those are probably the best format to use since it allows for the easiest configuring of the plots, and the publishers prefer it since its easy to access all the underlying plot information.

The increase in SVG size is due to the fact it has to contain a mathematical description of whatever is in the plot, and in the cast of a "highly complex" time series like a voltage trace, it is VERY complicated to perfectly represent that information. The ballooning size of SVGs is fundamental to the file format, and there are not quick fixes around this that are better than just using a bitmap like PNG to merely encode pixel information.

As an example, when I run the below code to get a PNG of the voltage waveforms,

```{matlab}
eqns={
  'dv/dt=Iapp+@current+noise*randn(1,N_pop); Iapp=0; noise=0'
  'monitor iGABAa.functions, iAMPA.functions'
};
% Tip: monitor all functions of a mechanism using: monitor MECHANISM.functions

time_end = 1000; % in milliseconds

% create DynaSim specification structure
s=[];
s.populations(1).name='E';
s.populations(1).size=80;
s.populations(1).equations=eqns;
s.populations(1).mechanism_list={'iNa','iK'};
s.populations(1).parameters={'Iapp',5,'gNa',120,'gK',36,'noise',40};
s.populations(2).name='I';
s.populations(2).size=20;
s.populations(2).equations=eqns;
s.populations(2).mechanism_list={'iNa','iK'};
s.populations(2).parameters={'Iapp',0,'gNa',120,'gK',36,'noise',10};


s.connections(1).direction='I->E';
s.connections(1).mechanism_list={'iGABAa'};
% connectivity matrix defined using a string that evalutes to a numeric matrix
s.connections(1).parameters={'tauD',10,'gGABAa',.1,'netcon','ones(N_pre,N_post)'};
s.connections(2).direction='E->I';
s.connections(2).mechanism_list={'iAMPA'};
 % connectivity set using a numeric matrix defined in script
s.connections(2).parameters={'tauD',2,'gAMPA',.1,'netcon',ones(80,20)};

%% Simulate Sparse Pyramidal-Interneuron-Network-Gamma (sPING)
data=dsSimulate(s,...
    'overwrite_flag',1, 'save_results_flag',1,...
    'save_data_flag',1,'tspan',[0 time_end],...
    'study_dir','exampleTest','verbose_flag',1,...
    'plot_functions',{@dsPlot},...
    'plot_options',{{'plot_type','waveform','format','png'}});
```

I get a PNG of size 128 KiB. But when I tell it to save this relatively simple voltage trace to SVG, I get a file of **a whopping 106 MiB**, which is almost 1000x larger!!

If anyone has any objections, let us know here. This seems to be affecting some of the newer students so I plan on changing it very soon.

**Comment by erik-roberts** [comment]
duplicate of #422 . closing latter.
